README.md

```diff
@@ -245,12 +245,21 @@ labelled as active.
 2. Create a new key
 3. Start the key rotation cycle
 
-#### Key rotation
+#### Key renewal
 
-Keys are automatically rotated. This can be configured on controller startup with
+Keys are automatically renewed. This can be configured on controller startup with
 the `--rotate-period=<value>` flag. The `value` field can be given as golang
```

Comment:
Please mention default value, in sensible units (ie don't want to run `bc -l` to convert to days/months).

Reply:
PTAL